### PR TITLE
Update RF24Gateway_ncurses.cpp

### PR DESCRIPTION
**Pull request by TMRh20**
Minor change to prevent segfault on non-master nodes

### CODE REVIEW

File: examples/ncurses/RF24Gateway_ncurses.cpp [hunk 1]
```diff
@@ -94,7 +94,7 @@ int main()
 {
 
     gw.begin();
-    mesh.setStaticAddress(8, 1);
+    //mesh.setStaticAddress(8, 1);
```
[the remark this answers is not in the document]
Comment (2bndy5) [reply examples/ncurses/RF24Gateway_ncurses.cpp]: This is the payoff for using the PR review feature of cpp-linter 🎉

**Comment by 2bndy5** [comment]
If you don't want the bot to spam the PR thread, then make the PR a "draft" instead of a normal PR that's ready for review.

image
or
image

**Comment by 2bndy5** [comment]
There seems to build errors about the net layers template specializations:

```
[ 50%] Building CXX object CMakeFiles/rf24gateway.dir/RF24Gateway.cpp.o
In file included from /home/runner/work/RF24Gateway/RF24Gateway/RF24Gateway/RF24Gateway.cpp:7:
/home/runner/work/RF24Gateway/RF24Gateway/RF24Gateway/RF24Gateway.h:43:7: error: using typedef-name ‘RF24Network’ after ‘class’
   43 | class RF24Network;
      |       ^~~~~~~~~~~
In file included from /home/runner/work/RF24Gateway/RF24Gateway/RF24Gateway/RF24Gateway.h:29,
                 from /home/runner/work/RF24Gateway/RF24Gateway/RF24Gateway/RF24Gateway.cpp:7:
/usr/arm-linux-gnueabihf/include/RF24Network/RF24Network.h:992:26: note: ‘RF24Network’ has a previous declaration here
  992 | typedef ESBNetwork<RF24> RF24Network;
      |                          ^~~~~~~~~~~
In file included from /home/runner/work/RF24Gateway/RF24Gateway/RF24Gateway/RF24Gateway.cpp:7:
/home/runner/work/RF24Gateway/RF24Gateway/RF24Gateway/RF24Gateway.h:44:7: error: using typedef-name ‘RF24Mesh’ after ‘class’
   44 | class RF24Mesh;
      |       ^~~~~~~~
In file included from /home/runner/work/RF24Gateway/RF24Gateway/RF24Gateway/RF24Gateway.h:30,
                 from /home/runner/work/RF24Gateway/RF24Gateway/RF24Gateway/RF24Gateway.cpp:7:
/usr/arm-linux-gnueabihf/include/RF24Mesh/RF24Mesh.h:384:41: note: ‘RF24Mesh’ has a previous declaration here
  384 | typedef ESBMesh<ESBNetwork<RF24>, RF24> RF24Mesh;
      |                                         ^~~~~~~~
gmake[2]: *** [CMakeFiles/rf24gateway.dir/build.make:76: CMakeFiles/rf24gateway.dir/RF24Gateway.cpp.o] Error 1
gmake[1]: *** [CMakeFiles/Makefile2:83: CMakeFiles/rf24gateway.dir/all] Error 2
gmake: *** [Makefile:156: all] Error 2
```
Not sure why this is happening now and wasn't before, but I'm stopping for now (need to sleep)...

**Comment by TMRh20** [comment]
Hehe, no rush on this one.

**Comment by 2bndy5** [comment]
I didn't realize this branch is based on 1.x branch instead of master (2.x). And the CI is using v2.x of RF24Network & RF24Mesh libs. Luckily, I have already foreseen this, so it was an easy fix in the CI yml file.

**Review by 2bndy5 — APPROVED** [review]

**Comment by TMRh20** [comment]
Yeah, I forgot what I was doing earlier and this had me a bit confused for a moment too.

**Pull request merged**